build_library/qemu_template.sh

```diff
@@ -158,7 +158,12 @@ else
         arm64-usr+aarch64)
             set -- -machine virt,accel=kvm,gic-version=3 -cpu host -smp "${VM_NCPUS}" -nographic "$@" ;;
         arm64-usr+*)
-            set -- -machine virt -cpu cortex-a57 -smp 1 -nographic "$@" ;;
+            if test "${VM_NCPUS}" -gt 4 ; then
+                VM_NCPUS=4
```

Comment:
If the actual number of CPUs is equal to 4, then the result `VM_NCPUS` becomes 2.
Isn't it supposed to be `-ge 4`?

Reply:
I think that's ok to use 2 if there are just 4 available.

Reply:
:+1: